pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
```

Comment:
In the toml we have a binding of: `torch = "^1.9.0"` btw.  I think that needs incremented since this is for 2.x minimum

Reply:
Done.